docs/source/guide/general/introduction.rst

```diff
@@ -0,0 +1,67 @@
+############
+Introduction
+############
+
+Repository Service for TUF
+##########################
+
+RSTUF Components
+================
+
+.. image:: /_static/2_1_rstuf.png
+
+Repository Service for TUF (RSTUF) is a combination of micro-services provided
+as containers:
+
+ * Repository Service for TUF API (RSTUF API ``repository-service-tuf-api``)
+ * Repository Service for TUF Worker (RSTUF Worker ``repository-service-tuf-worker``)
+
+Repository Service for TUF (RSTUF) also provides a Command Line Interface tool
+to manage your RSTUF deployment.
+
+ * Repository Service for TUF CLI (RSTUF CLI ``repository-service-tuf-cli``)
+
+Repository Service for TUF API (RSTUF API)
+------------------------------------------
+
+The :ref:`guide/repository-service-tuf-api/index:Repository Service for TUF API`
+(RSTUF API) servers an HTTP REST API for integration and service management.
```

Comment:
Do you mean to say that this "serves" an HTTP REST API ?  We might want to substitute the word "supplies" or "delivers".

Reply:
LGTM